src/script/lua_api/CMakeLists.txt

```diff
@@ -29,4 +29,5 @@ set(client_SCRIPT_LUA_API_SRCS
 	${CMAKE_CURRENT_SOURCE_DIR}/l_storage.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/l_sound.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/l_localplayer.cpp
+	${CMAKE_CURRENT_SOURCE_DIR}/l_particles_local.cpp
```

Comment:
And android? `build/android/jni/Android.mk` requires this file and `l_localplayer.cpp` aswell.
If you've got the time: could you please sort this list alphabetically?